python/pyspark/ml/linalg/__init__.py

```diff
@@ -746,11 +746,12 @@ def __hash__(self):
 class Vectors(object):
 
     """
-    Factory methods for working with vectors. Note that dense vectors
-    are simply represented as NumPy array objects, so there is no need
-    to covert them for use in MLlib. For sparse vectors, the factory
-    methods in this class create an MLlib-compatible type, or users
-    can pass in SciPy's C{scipy.sparse} column vectors.
+    Factory methods for working with vectors.
+
+    .. note:: Dense vectors are simply represented as NumPy array objects,
```

Comment:
This is another example where I think it works either way -- it isn't necessarily a stand-alone note, as much as a sentence that begins "Note that...", but it works OK either way in these cases.